reference/5.1/Microsoft.PowerShell.Core/About/about_Automatic_Variables.md

```diff
@@ -230,43 +213,35 @@ PS> $OFS="`n`n";[string]$a
 ```
 
 
-
 ### $PID
-Contains the process identifier (PID) of the process that is hosting the current  Windows PowerShell session.
+Contains the process identifier (PID) of the process that is hosting the current Windows PowerShell session.
 
 
 ### $PROFILE
-Contains the full path of the  Windows PowerShell profile for the current user and the current host application. You can use this variable to represent the profile in commands. For example, you can use it in a command to determine whether a profile has been created:
-
+Contains the full path of the Windows PowerShell profile for the current user and the current host application. You can use this variable to represent the profile in commands. For example, you can use it in a command to determine whether a profile has been created:
 
 ```
-test-path $profile
+Test-Path $Profile
 ```
 
-
 Or, you can use it in a command to create a profile:
 
-
 ```
-new-item -type file -path $pshome -force
+New-Item -Name $PROFILE -Type File -Path $PSHOME -Force
 ```
 
-
 You can also use it in a command to open the profile in Notepad:
 
-
 ```
-notepad $profile
+Notepad $PROFILE
```

Comment:
I know this is being pernickety but as notepad is an application shouldn't it really be in lowercase?

I only say this because that is what is returned from Get-Command notepad